Cargo.toml

```diff
@@ -180,6 +180,8 @@ codegen-units = 1
 [profile.production-stripped]
 inherits = "production"
 strip = true
+opt-level = "s"
+lto = true
```

Comment:
`lto = true` seems to always be a win (ignoring compile times) so perhaps we should make `production` have `lto = true`. 

Regarding the opt-level, I would be curious how `opt-level = 2` compares. 
Then again, the best opt-level likely also depends on the target architecture so probably the results will differ depending on what we test on.

Reply:
Agree. I remember comparing this as well, the results weren't as good as the options I endedup mentioning in the PR descriptions.  anyway here are the results 

| Category | opt-level-2 |
| -------- | -------- |
| FP (synthetic)     | 202.1ms     |
| FPs (synthetic)     | 202.1ms     |
| Renderer (synthetic)    | 97.77ms |
| Parse (synthetic)     | 5.809ms     |
| Layout (synthetic)     | 51.19ms     |
| Script (Synthetic)     | 7.974ms     |
| Rasterise (synthetic)     | 15.40ms     |
| Compositing (Real)     |   15.41ms |
| LayoutPerform (Real)     | 51.19ms       |
| ScriptParseHTML (Real)     | 5.809ms     |
| ScriptEvaluate (Real)     | 7.988ms     |